src/DataAccess/Entities/Apprenticeship/Apprenticeship.cs

```diff
@@ -32,5 +32,6 @@ public Apprenticeship()
         public long Ukprn { get; set; }
         public List<StartDateChange> StartDateChanges { get; set; }
         public string? TrainingCourseVersion { get; set; }
+        public bool? PaymentsFrozen { get; set; }
     }
 }
```

Comment:
Could change to filescoped namespace

Reply:
done, also updated Approval so all the entities are in line